apps/agreement/contracts/arbitration/IArbitrable.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity ^0.4.24;
+
+import "@aragon/os/contracts/lib/standards/ERC165.sol";
+
+import "./IArbitrator.sol";
+
+
+/**
+* @title Arbitrable interface
+* @dev This interface is the one extended by `IAgreement` so it can be used by `IArbitrator`, its dispute resolution protocol.
+*      This interface was manually-copied from https://github.com/aragon/aragon-court/blob/v1.1.3/contracts/arbitration/IArbitrable.sol
+*      since we are using different solidity versions.
+*/
+contract IArbitrable is ERC165 {
+    bytes4 internal constant ERC165_INTERFACE_ID = bytes4(0x01ffc9a7);
+    bytes4 internal constant ARBITRABLE_INTERFACE_ID = bytes4(0x88f3ee69);
+
+    /**
+    * @dev Emitted when an IArbitrable instance's dispute is ruled by an IArbitrator
+    * @param arbitrator IArbitrator instance ruling the dispute
+    * @param disputeId Identification number of the dispute being ruled by the arbitrator
+    * @param ruling Ruling given by the arbitrator
+    */
+    event Ruled(IArbitrator indexed arbitrator, uint256 indexed disputeId, uint256 ruling);
+
+    /**
+    * @dev Emitted when new evidence is submitted for the IArbitrable instance's dispute
+    * @param arbitrator IArbitrator submitting the evidence for
+    * @param disputeId Identification number of the dispute receiving new evidence
+    * @param submitter Address of the account submitting the evidence
+    * @param evidence Data submitted for the evidence of the dispute
+    * @param finished Whether or not the submitter has finished submitting evidence
+    */
+    event EvidenceSubmitted(IArbitrator indexed arbitrator, uint256 indexed disputeId, address indexed submitter, bytes evidence, bool finished);
+
+    /**
+    * @dev Submit evidence for a dispute
+    * @param _disputeId Id of the dispute in the Court
+    * @param _evidence Data submitted for the evidence related to the dispute
+    * @param _finished Whether or not the submitter has finished submitting evidence
+    */
+    function submitEvidence(uint256 _disputeId, bytes _evidence, bool _finished) external;
+
+    /**
+    * @dev Give a ruling for a certain dispute, the account calling it must have rights to rule on the contract
+    * @param _disputeId Identification number of the dispute to be ruled
+    * @param _ruling Ruling given by the arbitrator, where 0 is reserved for "refused to make a decision"
+    */
+    function rule(uint256 _disputeId, uint256 _ruling) external;
+
+    /**
+    * @dev ERC165 - Query if a contract implements a certain interface
+    * @param _interfaceId The interface identifier being queried, as specified in ERC-165
+    * @return True if this contract supports the given interface, false otherwise
+    */
+    function supportsInterface(bytes4 _interfaceId) public pure returns (bool) {
+        return _interfaceId == ARBITRABLE_INTERFACE_ID || _interfaceId == ERC165_INTERFACE_ID;
```

Comment:
Should we follow the `super()` pattern here given that we're using aragonOS' `ERC165`?